src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ CHANGELOG
 3.0.0
 -----
 
- * removed all deprecated codes from 2.x versions
+ * removed all deprecations from 2.x versions
```

Comment:
the previous message is better IMO. We removed the deprecate code, not just the deprecations themselves (which would mean un-deprecating the code)